lib/ipam/ipam.go

```diff
@@ -207,12 +207,12 @@ func (c ipamClient) getBlockFromAffinity(ctx context.Context, aff *model.KVPair)
 // If any requested pool does not exist, or is not enabled, an error is returned.
 // If no pools are requested, all enabled pools are returned.
 // Also applies selector logic on node labels to determine if the pool is a match.
```

Comment:
We should update the comment here to indicate it also returns the full set of enabled IP pools